src/evergreen/evg_config.rs

```diff
@@ -5,7 +5,7 @@ use shrub_rs::models::{project::EvgProject, task::EvgTask, variant::BuildVariant
 
 const REQUIRED_PREFIX: &str = "-required";
 
-pub trait EvgConfigService {
+pub trait EvgConfigService: Sync + Send {
```

Comment:
`Send` and `Sync` have to do with multi-threading, which we are not using yet, but we will be soon. There are details in [the rust book](https://doc.rust-lang.org/book/ch16-04-extensible-concurrency-sync-and-send.html).